DTAdminTests/TestDataManager.swift

```diff
@@ -0,0 +1,365 @@
+//
+//  TestDataManager.swift
+//  DTAdminTests
+//
+//  Created by Володимир on 04.11.17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import XCTest
+@testable import DTAdmin
+
+
+
+class TestDataManager: XCTestCase {
+    var sessionUnderTest: URLSession!
+    override func setUp() {
+        super.setUp()
+        CommonNetworkManager.shared().logIn(username: "admin", password: "dtapi_admin") { (user, error) in
```

Comment:
I really appreciate this effort. However it is not unit tests. The unit tests cannot be dependent on backend. It's like integration tests and IMO its better to use mock responses in this case